index.js

```diff
@@ -95,19 +164,11 @@ getMove = function (control) {
 };
 
 module.exports = defineProperties(getFn(), {
-	width: d.gs(process.stdout.getWindowSize ? function () {
-		return process.stdout.getWindowSize()[0];
-	} : function () {
-		return tty.getWindowSize ? tty.getWindowSize()[1] : 0;
-	}),
-	height: d.gs(getHeight = process.stdout.getWindowSize ? function () {
-		return process.stdout.getWindowSize()[1];
-	} : function () {
-		return tty.getWindowSize ? tty.getWindowSize()[0] : 0;
-	}),
-	reset: d.gs(function () {
-		return repeat.call('\n', getHeight() - 1) + '\x1bc';
-	}),
+	trim: d(trim),
+	throbber: d(throbber),
+	width: d.gs(function () { return process.stdout.columns || 0; }),
+	height: d.gs(function () { return process.stdout.rows || 0; }),
+	reset: d.gs(function () { return '\n\x1bc'; }),
```

Comment:
node v0.12 does not have `process.stdout.getWindowSize`, but have `process.stdout.columns` (and rows).
v0.10 have both `process.stdout.getWindowSize` and `process.stdout.columns`. The earlier versions might have only `process.stdout.getWindowSize`.
I think this is the place for graceful degradation.